scripts/download-cred-helpers.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# This script downloads the gptscript-credential-helpers. (For use in CI.)
+
+GPTSCRIPT_CRED_HELPERS_VERSION="v0.1.0"
+BINARY_DIR="binaries"
+
+mkdir -p "$BINARY_DIR"
+cd "$BINARY_DIR"
+
+wget -O gptscript-credential-osxkeychain "https://github.com/gptscript-ai/gptscript-credential-helpers/releases/download/${GPTSCRIPT_CRED_HELPERS_VERSION}/gptscript-credential-osxkeychain"
```

Comment:
Why wget :)

Reply:
I actually like wget a lot just for downloading files, lol. I can never remember the arguments I need to give to curl. 🤷‍♂️